R/count-tally.R

```diff
@@ -151,7 +151,12 @@ add_tally <- function(x, wt, sort = FALSE) {
     n <- quo(sum(!!wt, na.rm = TRUE))
   }
 
-  n_name <- n_name(tbl_vars(x))
+  n_name <- n_name(tbl_vars(x), name)
```

Comment:
```suggestion
  n_name <- n_name(group_vars(x), name)
```

same as above, would deal with #3838